config_test.go

```diff
@@ -474,6 +474,64 @@ func TestParseCli(t *testing.T) {
 				"b2 concurrent connections must be more than 0 (current 0)",
 				"",
 			},
+			{
+				[]string{"--delete-uploaded", "yes"},
+				options{
+					Directory:               "/var/backups/postgresql",
+					Format:                  'c',
+					DirJobs:                 1,
+					CompressLevel:           -1,
+					Jobs:                    1,
+					PauseTimeout:            3600,
+					PurgeInterval:           -30 * 24 * time.Hour,
+					PurgeKeep:               0,
+					SumAlgo:                 "none",
+					CfgFile:                 "/etc/pg_back/pg_back.conf",
+					TimeFormat:              timeFormat,
+					Encrypt:                 false,
+					CipherPassphrase:        "",
+					WithRolePasswords:       true,
+					Upload:                  "none",
+					Download:                "none",
+					ListRemote:              "none",
+					AzureEndpoint:           "blob.core.windows.net",
+					B2ConcurrentConnections: 5,
+					DeleteUploaded:          true,
+				},
+				false,
+				false,
+				"",
+				"",
+			},
+			{
+				[]string{"--delete-uploaded", "true"},
+				options{
+					Directory:               "/var/backups/postgresql",
+					Format:                  'c',
+					DirJobs:                 1,
+					CompressLevel:           -1,
+					Jobs:                    1,
+					PauseTimeout:            3600,
+					PurgeInterval:           -30 * 24 * time.Hour,
+					PurgeKeep:               0,
+					SumAlgo:                 "none",
+					CfgFile:                 "/etc/pg_back/pg_back.conf",
+					TimeFormat:              timeFormat,
+					Encrypt:                 false,
+					CipherPassphrase:        "",
+					WithRolePasswords:       true,
+					Upload:                  "none",
+					Download:                "none",
+					ListRemote:              "none",
+					AzureEndpoint:           "blob.core.windows.net",
+					B2ConcurrentConnections: 5,
+					DeleteUploaded:          true,
+				},
+				false,
+				false,
+				"invalid value for --delete-uploaded: value must be \"yes\" or \"no\"",
+				"",
+			},
```

Comment:
@orgrim  I've added these two tests.

I'm not sure of the quality of my implementation.

Does it suit you?